Code/GraphMol/Fingerprints/FingerprintGenerator.h

```diff
@@ -15,31 +15,58 @@
 #include <DataStructs/SparseIntVect.h>
 #include <DataStructs/ExplicitBitVect.h>
 #include <DataStructs/SparseBitVect.h>
+#include <vector>
+#include <memory>
 #include <cstdint>
 
 namespace RDKit {
 class ROMol;
 
 struct RDKIT_FINGERPRINTS_EXPORT AdditionalOutput {
-  // will review this structure once more fingerprint types are implemented
+  using atomToBitsType = std::vector<std::vector<std::uint64_t>>;
+  using bitInfoMapType =
+      std::map<std::uint64_t,
+               std::vector<std::pair<std::uint32_t, std::uint32_t>>>;
```

Comment:
Probably not, but just to make sure, should these be updated to `uint64_t` too ?

Reply:
unfortunately not